Examples/Scripts/Python/truth_tracking_gx2f.py

```diff
@@ -33,18 +34,23 @@ def runTruthTrackingGx2f(
     )
 
     s = s or acts.examples.Sequencer(
-        events=10000, numThreads=-1, logLevel=acts.logging.INFO
+        events=100, numThreads=-1, logLevel=acts.logging.INFO
     )
 
-    rnd = acts.examples.RandomNumbers()
+    rnd = acts.examples.RandomNumbers(seed=42)
     outputDir = Path(outputDir)
 
     if inputParticlePath is None:
         addParticleGun(
             s,
+            ParticleConfig(num=1, pdg=acts.PdgParticle.eMuon, randomizeCharge=True),
+            EtaConfig(-2.5, 2.5, uniform=True),
             MomentumConfig(100.0 * u.GeV, 100.0 * u.GeV, transverse=True),
```

Comment:
These two are different from the KF one because of the FPE?

Alternatively we could suppress the FPE in this job until it is fixed.

Reply:
exactly, since there is something going on outside that range, I don't want to just suppress it in the output.